packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -201,21 +201,17 @@ export function FileSnapshotWriterClassFactory<TBase extends ReaderConstructor>(
             delete tree.id;
             removeNullTreIds(tree);
 
-            if (ref) {
-                this.commitsWriter[commitName] = tree;
-            } else {
-                // Rebuild latest tree - runtime will ask for it when generating next snapshot to write out
-                // non-changed commits for data stores
-                await this.writeOutFullSnapshot(tree);
+            // Rebuild latest tree - runtime will ask for it when generating next snapshot to write out
+            // non-changed commits for data stores
+            await this.writeOutFullSnapshot(tree);
```

Comment:
This does not look right (at least based on how this code worked before).
In old "snapshot" world, we committed all data stores (as individual commits) and then would write out container tree (that would reference data store commits). ref being empty (or undefined?) was a way to know it's container. This implementation was not writing out individual commits (data stores), it was accumulating them in memory to write out single file accumulating whole container.

Reply:
This will not break UTs, but I believe it will make them almost no-op, as we are loosing (not including) data store snapshots into full container snapshot for future comparison.

Reply:
Reverted in latest